app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -15,7 +15,7 @@ export const MyMinisterSettings: React.FC = () => {
 
   let activeMinisters = ministers
     .filter((m) => m.isEnabled)
-    .sort((a, b) => a.name.localeCompare(b.name));
+    .sort((a, b) => a.sortOrder - b.sortOrder || a.name.localeCompare(b.name));
```

Comment:
Won't this ignore `sortOrder` if `=-1 || 0` and then use the `name` instead?  The `sortOrder` should be the priority sort`.

Reply:
I honestly wasn't too sure how that worked either, but it does..
![image](https://github.com/bcgov/tno/assets/63015960/94f71b89-6476-4d94-b396-928d0b4c63fa)

![image](https://github.com/bcgov/tno/assets/63015960/a65f0e49-495b-42ba-813b-2125e09b3ee5)